test/inject/inject_embed_test.dart

```diff
@@ -8,24 +8,23 @@ import 'dart:html';
 import 'package:test/test.dart';
 import 'package:dart_pad/inject/inject_embed.dart' as inject_embed;
 
+// todo: determine how to load embed-new-flutter.html and other assets.
```

Comment:
super nit: `embed-flutter.html`? Also, it'd be good to throw your ldap in there if this is going to stay.